### PR TITLE
Clustered execution is supported

### PR DESCRIPTION
**Pull request by ppcano**

### CODE REVIEW

**Comment by ppcano** [comment]
cc @yorugac

File: src/data/markdown/translated-guides/en/01 Getting started/03 Running k6.md [hunk 1]
```diff
@@ -178,26 +178,50 @@ export default function () {
...
-## Running cloud tests
+## Execution modes
 
-k6 supports three test-execution modes:
+k6 supports three execution modes to run a k6 test: local, distributed, and cloud. 
 
-- [Local](#running-local-tests): on your local machine or a CI server.
-- [Cloud](/cloud): on cloud infrastructure managed by k6 Cloud.
-- Clustered: on more than one machine managed by you. [Not supported yet](https://github.com/grafana/k6/issues/140).
+> One of the k6 design goals is to allow switching the execution modes with minimal changes.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/translated-guides/en/01 Getting started/03 Running k6.md:185]:
```suggestion
> **The design of k6 lets users switch between execution modes with minimal changes.**
```

File: src/data/markdown/translated-guides/en/01 Getting started/03 Running k6.md [hunk 2]
```diff
@@ -178,26 +178,50 @@ export default function () {
...
-k6 supports three test-execution modes:
+k6 supports three execution modes to run a k6 test: local, distributed, and cloud. 
 
-- [Local](#running-local-tests): on your local machine or a CI server.
-- [Cloud](/cloud): on cloud infrastructure managed by k6 Cloud.
-- Clustered: on more than one machine managed by you. [Not supported yet](https://github.com/grafana/k6/issues/140).
+> One of the k6 design goals is to allow switching the execution modes with minimal changes. 
+> &nbsp;
+> 
+> The primary goal is to support **running the test in the various execution modes without modifying the k6 script**. This ability will enable reusing your tests - no matter the environment and context of your tests.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/translated-guides/en/01 Getting started/03 Running k6.md:188]:
```suggestion
> This lets users reuse tests&mdash;no matter the environment and context. Making tests portable across multiple modes of execution is a key design goal of k6.
```

File: src/data/markdown/translated-guides/en/01 Getting started/03 Running k6.md [hunk 3]
```diff
@@ -178,26 +178,50 @@ export default function () {
...
-2. Log in to your account via the CLI.
-3. Use the `k6 cloud` command to run the script you already have.
+  ```bash
+  k6 run script.js
+  ```
 
-<CodeGroup labels={["Running a cloud test"]}>
+2. [Distributed](https://github.com/grafana/k6-operator): the test execution is distributed across a Kubernetes cluster. 
+  
+  <CodeGroup labels={["Running", "k6-resource.yml"]} lineNumbers={[true]}>
```
Comment (MattDodsonEnglish) [inline src/data/markdown/translated-guides/en/01 Getting started/03 Running k6.md:198]:
```suggestion
  <CodeGroup labels={["Running", "k6-resource.yaml"]} lineNumbers={[true]}> 
```

File: src/data/markdown/translated-guides/en/01 Getting started/03 Running k6.md [hunk 4]
```diff
@@ -178,26 +178,50 @@ export default function () {
...
-- [Local](#running-local-tests): on your local machine or a CI server.
-- [Cloud](/cloud): on cloud infrastructure managed by k6 Cloud.
-- Clustered: on more than one machine managed by you. [Not supported yet](https://github.com/grafana/k6/issues/140).
+> One of the k6 design goals is to allow switching the execution modes with minimal changes. 
+> &nbsp;
+> 
+> The primary goal is to support **running the test in the various execution modes without modifying the k6 script**. This ability will enable reusing your tests - no matter the environment and context of your tests.
 
-k6 has a goal of letting you run a test in all three execution modes without modifying the script.
+1. [Local](#running-local-tests): the test execution is run entirely on a single machine, container, or CI server.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/translated-guides/en/01 Getting started/03 Running k6.md:190]:
```suggestion
1. [Local](#running-local-tests): the test runs entirely on a single machine, container, or CI server. 
```

File: src/data/markdown/translated-guides/en/01 Getting started/03 Running k6.md [hunk 5]
```diff
@@ -178,26 +178,50 @@ export default function () {
...
-<CodeGroup labels={["Running a cloud test"]}>
+2. [Distributed](https://github.com/grafana/k6-operator): the test execution is distributed across a Kubernetes cluster. 
+  
+  <CodeGroup labels={["Running", "k6-resource.yml"]} lineNumbers={[true]}> 
 
-```bash
-$ k6 cloud script.js
-```
+  ```bash
+  kubectl apply -f /path/k6-resource.yml
```
Comment (MattDodsonEnglish) [inline src/data/markdown/translated-guides/en/01 Getting started/03 Running k6.md:201]:
```suggestion
  kubectl apply -f /path/k6-resource.yaml
```

File: src/data/markdown/translated-guides/en/01 Getting started/03 Running k6.md (hunk 4 above)
Comment (MattDodsonEnglish) [inline src/data/markdown/translated-guides/en/01 Getting started/03 Running k6.md:190]:
```suggestion
1. [Local](#running-local-tests): the test execution happens entirely on a single machine, container, or CI server. 
```

File: src/data/markdown/translated-guides/en/01 Getting started/03 Running k6.md [hunk 6]
```diff
@@ -178,26 +178,50 @@ export default function () {
...
+    parallelism: 4
+    script:
+      configMap:
+        name: "k6-test"
+        file: "script.js"
+  ```
+
+  </CodeGroup>
+
+3. [Cloud](/cloud): the test execution is run on the k6 Cloud.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/translated-guides/en/01 Getting started/03 Running k6.md:220]:
```suggestion
3. [Cloud](/cloud): the test execution happens on k6 Cloud.  
```

File: src/data/markdown/translated-guides/en/01 Getting started/03 Running k6.md [hunk 7]
```diff
@@ -178,26 +178,50 @@ export default function () {
...
+
+  </CodeGroup>
+
+3. [Cloud](/cloud): the test execution is run on the k6 Cloud.  
+
+  ```bash
+  k6 cloud script.js
+  ```
+
+  Additionally, k6 Cloud can run cloud tests on your [own cloud infrastructure](/cloud/cloud-faq/private-load-zones/), and accepts the test results from a [local](/results-visualization/cloud/) or [distributed test](https://github.com/grafana/k6-operator#k6-cloud-output).
```
Comment (MattDodsonEnglish) [inline src/data/markdown/translated-guides/en/01 Getting started/03 Running k6.md:226]:
```suggestion
  Additionally, k6 Cloud can run cloud tests on your [own cloud infrastructure](/cloud/cloud-faq/private-load-zones/), and accept the test results from a [local](/results-visualization/cloud/) or [distributed test](https://github.com/grafana/k6-operator#k6-cloud-output).
```

File: src/data/markdown/translated-guides/en/07 Misc/06 Archive.md [hunk 8]
```diff
@@ -80,12 +80,11 @@ from the k6 command-line via the `k6 cloud script.js` command (similar to `k6 ru
...
-### Clustered Execution (_future_)
+### Distributed Execution 
 
-In the future (see [our roadmap](https://github.com/grafana/k6/wiki/Roadmap)) k6 will
-support a clustered execution mode that will enable tests to be run across more than one
-node. This mode of execution is also likely to make use of the archive functionality to
-distribute the test files to all participating nodes.
+[k6-operator](https://github.com/grafana/k6-operator#multi-file-tests) can distribute a k6 test across a Kubernetes cluster. 
+
+When having a k6 test with multiple files, you can use the archive functionality to bundle the k6 test in a single "archived" file and pass this file to run the test.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/translated-guides/en/07 Misc/06 Archive.md:87]:
```suggestion
When a k6 test has multiple files, you can use the archive functionality to bundle the k6 test in a single *archived* file and pass this file to run the test.
```

**Review by MattDodsonEnglish — COMMENTED** [review]
I think it looks good though I suggested a few language changes:
- saying "execution runs" seemed a little awkward to me
- Turning `yml` to `yaml` to appease the linter.

Just to be clear, currently the only type of distributed test that k6 supports is with Kubernetes, right? I think it's okay to call the overall execution mode category  "distributed" for now, even though it applies only to a Kubernetes cluster, as k6 may add more distributed options in the future.

**Review by MattDodsonEnglish — APPROVED** [review]

**Pull request merged**